### PR TITLE
Adding option strategies support

### PR DESCRIPTION
**Pull request by 4rumprom**
Extending functionalities to enable option trading strategies supported by Schwabs.

Adding to the Schwab class:
- API call function to pull option chain quotes for an underlying symbol. This optionally includes option greeks.
- Specific option trading function to create 2 to 4 legged strategies such as spreads, butterflies, condors etc.

Also included an example file that pulls the option chains, helps normalize the json response into a pandas dataframe and creates a dry run on an option straddle strategy.

### CODE REVIEW

File: schwab_api/schwab.py [hunk 1]
```diff
@@ -571,6 +765,39 @@ def get_lot_info_v2(self, account_id, security_id):
...
+        
+             ticker (str) - ticker of the underlying security
+             greeks (bool) - if greeks is true, you will also get the option greeks (Delta, Theta, Gamma etc... )
+        """
+        expiration_date = []
+        #for date in ExpirationDates:
+            # using python format to remove leading 0 for month and day.
+            # datetime.strftime leaves leading 0 and removing them is different for Unix and Windows.
+        #    formatted_date = '{dt.month}/{dt.day}/{dt.year}'.format(dt = date)
+        #    expiration_date.append(formatted_date)
```
Comment (itsjafer) [inline schwab_api/schwab.py:782]: Should this comment be here?

File: example/example_options.py [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+
+# Login using playwright
+print("Logging into Schwab")
+logged_in = api.login(
+    username=username,
+    password=password,
+    totp_secret=totp_secret # Get this using itsjafer.com/#/schwab.
+)
+
+#get quotes for options.
```
Comment (itsjafer) [inline example/example_options.py:24]: Nit: Could we maintain a consistent style for comments (as in example.py)? i.e:

* Space beginning each comment
* Starting with a capital letter

File: example/example_options.py [hunk 3]
```diff
@@ -0,0 +1,83 @@
...
+OptionChain = api.get_options_chains_v2('$RUT') #try also with parameter greeks = True
```
Comment (itsjafer) [inline example/example_options.py:25]: nit: could we call this `option_chain`? `OptionChain` implies it is a class

File: schwab_api/schwab.py [hunk 4]
```diff
@@ -385,6 +387,198 @@ def trade_v2(self,
...
+                        210 - Butterfly put spread
+                        218 - custom 3 legs
+                        4 leg strategies:
+                        211 - condor call spread
+                        212 - condor put spread
+                        214 - iron condor
+                        219 - custom 4 legs
+                        Combinations:
+                        226 - straddle
+                        227 - strangle
```
Comment (itsjafer) [inline schwab_api/schwab.py:433]: I wonder if we might want to consider turning these into a class-wide enumerator; though we can push that to a later CL.

**Review by itsjafer — COMMENTED** [review]
Thank you so much for the contribution! I left a few nitpick comments!

File: example/example_options.py (hunk 2 above)
Comment (4rumprom) [reply example/example_options.py:24]: Sure

File: schwab_api/schwab.py (hunk 1 above)
Comment (4rumprom) [reply schwab_api/schwab.py:782]: Probably not 😂

File: example/example_options.py (hunk 3 above)
Comment (4rumprom) [reply example/example_options.py:25]: No pb. Will do

File: schwab_api/schwab.py (hunk 4 above)
Comment (4rumprom) [reply schwab_api/schwab.py:433]: I thought the same thing. Might make sense for trades as well like market order, limit order etc

Comment (4rumprom) [reply schwab_api/schwab.py:433]: But I wanted to get a working version out first and it can always be refined.

**Review by itsjafer — APPROVED** [review]

**Pull request merged**